.github/workflows/tests.yml

```diff
@@ -38,7 +38,7 @@ jobs:
 
       - name: flake8
         working-directory: ./scripts
-        run: flake8 ./*.py
+        run: flake8 --ignore E402,W504
```

Comment:
Could you put these flags/rules into a config file maybe? For example in a `setup.cfg` file in the root directory under header `[flake8]`?

Reply:
On the other hand though, do we really need to have an import that somewhere other than the top of the file? Smells like bad code design. In any case, we should most certainly not ignore [`E402`](https://www.flake8rules.com/rules/E402.html) warnings across the entire project. If we really can't (easily) avoid it, just ignore it inline in the place where that warning is raised (I hope it's only one).

Similarly, I'm pretty sure [`W504`](https://www.flake8rules.com/rules/W504.html) can always be avoided?